src/storage/mvcc/reader/scanner/backward.rs

```diff
@@ -266,6 +264,16 @@ impl<S: Snapshot> BackwardKvScanner<S> {
                     if self.met_newer_ts_data == NewerTsCheckState::NotMetYet {
                         self.met_newer_ts_data = NewerTsCheckState::Met;
                     }
+                    if self.cfg.isolation_level == IsolationLevel::RcCheckTs {
```

Comment:
Is it ok to skip this record if the `WriteType` is `Lock` or `Rollback`?

Reply:
I think so, I'll add more tests to cover this path.

Reply:
It's a bit complext to check if there's a newer version which is not `Lock` and `Rollback`. As the feature is recommanded to be used where there's less read-write conflict, maybe it's ok to return error and let client retry directly.

Reply:
May left a TODO here?

Reply:
Ok.